supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -137,19 +137,21 @@ If available, inform users of alternative capabilities and workarounds.
 [discrete]
 ===== Deprecation notice
 ----
-In <product> <release>, <name of capability> is deprecated. Bug fixes and support are provided through the end of the <releasename or, if unknown, “a future”> life cycle. After which, no new feature enhancements are made. (Optional: You can use <alternative capability> instead.)
+In <product> <release>, <name of capability or feature> is deprecated and is planned to be removed in the <deprecation time frame>. Bug fixes and support are provided through the end of the current release life cycle, but be aware that deprecated items do not receive further enhancements and will be removed. [Optional, if an alternative is available] Alternatively, you can use <alternative capability or feature> instead.
 ----
+When citing deprecation time frames, refer to specific releases (such as "the next release") only if that time frame is known to be accurate. Otherwise, refer to "a future release" because it gives flexibility to account for changes to the planned deprecation.
+
 .Example deprecation notice doc text
 ----
-In Red Hat Openstack Platform (RHOSP) 14, the director graphical user interface is deprecated. Bug fixes and support is provided through the end of a future life cycle but Red Hat do not implement new feature enhancements.
+In Red Hat Openstack Platform (RHOSP) 14, the director graphical user interface is deprecated and is planned to be removed in the next release. Bug fixes and support are provided through the current release life cycle, but be aware that deprecated items do not receive enhancements and will be removed.
 ----
 
 [discrete]
 ===== Removal notice
 ----
-In <product> <release>, <name of capability> is removed. Bug fixes and support is provided through the end of the <releasename or, if unknown, “a future”> life cycle. After which, no new feature enhancements are made. (Optional: You can use <alternative capability> instead.)
+In <product> <current release>, <name of capability or feature> is removed. Bug fixes and support are provided only through the end of the <previous release> life cycle. [Optional, if an alternative capability or feature is available] Alternatively, you can use <alternative capability or feature> instead. 
```

Comment:
Place user-replaced values in italics. 

See: https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values

Reply:
[Optional, if an alternative capability or feature is available] Alternatively, you can use <alternative capability or feature> instead. 

New paragraph 

> Optional:You can use `<alternative capability or feature>` instead.

Reply:
The `Optional:` is usually for an optional step in a procedure, not sure if we need that here since these notices read like paragraphs, maybe it's fine to keep them that way, wdyt @dfitzmau ?

Reply:
@ndeevy @dfitzmau I will remove the Optional portion of the example. I think people are smart enough to know that if there's no alternative available, then it can't be listed.